roles/container-engine/meta/main.yml

```diff
@@ -23,6 +23,13 @@ dependencies:
       - container-engine
       - crun
 
+  - role: container-engine/youki
```

Comment:
this should probably be conditioned by a supported container manager, as far as I can tell from the project's public documentation they only support cri-o.